server/build/babel/preset.js

```diff
@@ -11,19 +11,20 @@ const envPlugins = {
 
 const plugins = envPlugins[process.env.NODE_ENV] || envPlugins['development']
 
-module.exports = {
+module.exports = (context, opts = {}) => ({
   presets: [
-    [require.resolve('babel-preset-env'), {
+    [require.resolve('babel-preset-env'), Object.assign({}, opts['preset-env'], {
```

Comment:
Lets do this:
```
{modules: false, ...opts['preset-env']}
```
So that people don't forget to pass off the modules to webpack. I can totally see users make this mistake 😄, or well, you already made this mistake in the example. Nothing bad about good defaults 👌 And if the users want to override, that will just work totally fine 👌 👍 💯

Reply:
Ah - okay, yep!

Reply:
🤔 Hm, is my code not doing that, just using Object.assign instead of spread notation? I'll switch to spread notation, but I suspect this might already be correct.

Reply:
Oh, you're right, totally missed it 😅 Either way, changing it around allows the user to override it if they really want to 😄